.github/workflows/ci-build.yml

```diff
@@ -159,7 +159,7 @@ jobs:
           export WASMEDGE_BUILD_DIR="$(pwd)/WasmEdge/build"
           export WASMEDGE_PLUGIN_PATH="$(pwd)/WasmEdge/build/plugins/wasmedge_process"
           export LD_LIBRARY_PATH="$(pwd)/WasmEdge/build/lib/api"
-          cargo test --workspace --locked --features aot,async,wasmedge_process,ffi -- --nocapture --test-threads=1
+          cargo test --workspace --locked --features aot,async,wasmedge_process,ffi -- --nocapture --test-threads=1 --skip test_vmbuilder
```

Comment:
Why skip VmBuilder-related tests here?

Reply:
Because Fedora cannot install the rustls plugin.

Reply:
Because Fedora cannot install the rustls plugin.